builds/templates/install-dotnet2.yaml

```diff
@@ -3,4 +3,4 @@ steps:
     displayName: Install .NET Core sdk
     inputs:
       packageType: sdk
-      version: 2.1.806
+      version: 2.2.207
```

Comment:
why change this?

Reply:
I was thinking that if this works on a more up-to-date version then it is better